erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -765,7 +765,10 @@ def get_events(start, end, filters=None):
 	return data
 
 @frappe.whitelist()
-def make_purchase_order_for_drop_shipment(source_name, for_supplier=None, target_doc=None):
+def make_purchase_order_for_drop_shipment(source_name, for_supplier=None, selected_items=[], target_doc=None):
```

Comment:
As now PO can be made against any item in SO, remove the drop_shipment term from function naming

Reply:
done